spinalcordtoolbox/utils/sys.py

```diff
@@ -301,18 +301,23 @@ def run_proc(cmd, verbose=1, raise_exception=True, cwd=None, env=None, is_sct_bi
         cwd = os.getcwd()
 
     if env is None:
-        env = os.environ
+        env = os.environ.copy()
 
     if is_sct_binary:
+        # create an absolute path to the binary inside of SCT's bin directory
         name = cmd[0] if isinstance(cmd, list) else cmd.split(" ", 1)[0]
         path = os.path.join(__bin_dir__, name)
-
         if isinstance(cmd, list):
             cmd[0] = path
         elif isinstance(cmd, str):
             rem = cmd.split(" ", 1)[1:]
             cmd = path if len(rem) == 0 else "{} {}".format(path, rem[0])
 
+        # also, for windows, add the bin directory to the path (to allow ANTs to access 'msvc-runtime' DLLs)
+        # see also: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4655#issuecomment-2430178901
+        if sys.platform.startswith("win32"):
+            env['PATH'] = env['PATH'] + os.pathsep + __bin_dir__
```

Comment:
Maybe put our `__bin_dir__` at the front, to give it priority? This would be consistent with a few lines above, where we force the command to be from `__bin_dir__` rather than relying on the `PATH`.
```suggestion
            env['PATH'] = __bin_dir__ + os.pathsep + env['PATH']
```

Reply:
I think I originally put our bin dir last because I wanted to minimize the effects of this change, and only supply what was missing (rather than overriding DLLs that were previously available on the PATH).

But, I guess if our goal is to isolate the SCT env from the rest of the system, then relying on files from our own venv is preferable to relying on system files anyway? 

I imagine it's the sort of thing that will be hard to predict the impact of until we see it in action. But, regardless, I think this is an okay change to make given the limited conditions where this applies.